src/Cache/Cache.php

```diff
@@ -368,6 +368,7 @@ public static function writeMany($data, $config = 'default')
      */
     public static function read($key, $config = 'default')
     {
+        // In 4.x this needs to change to use pool()
```

Comment:
This is actually a "TODO", isn't it?

Reply:
It *is* except our lint rules will fail the build on TODOs being present, and I'm frankly too lazy to change them.

Reply:
@markstory I thought so ;-)

Would you want to accept TODOs and/or FIXMEs in code during a fast changing development phase like for 3.next and/or 4.x?

Reply:
I'm fine with it, makes finding future tasks easier to grep for.

Reply:
I, personally, love using TODOs in my and our applications' code. But I also understand that you don't want to clutter your code with it.

Reply:
Yeah there's a balance to be had and I feel we might be too far on one side right now.

Reply:
> It is except our lint rules will fail the build on TODOs being present, and I'm frankly too lazy to change them.

I changed that for you with a simple exclusion of this sniff with a comment.